src/utilities/_hypre_utilities.hpp

```diff
@@ -95,7 +95,7 @@ using hypre_DeviceItem = void*;
 #endif
 
 #define CUSPARSE_NEWAPI_VERSION 11000
-#define CUSPARSE_NEWSPMM_VERSION 11201
+#define CUSPARSE_NEWSPMM_VERSION 11401
 #define CUDA_MALLOCASYNC_VERSION 11020
```

Comment:
I have no idea how these numbers come about, so hopefully someone else has.

Reply:
`11401` refers to the cuSPARSE version in which a new algorithm (CUSPARSE_SPMM_CSR_ALG3) was added for Sparse Matrix - Matrix Multiplication (cusparseSpMM). That algorithm shows better performance for small matrices. cuSPARSE `11401` was introduced in CUDA `11.2.1`, see: https://docs.nvidia.com/cuda/archive/11.2.1/cuda-toolkit-release-notes/index.html